metrics/web/test_driver/app_test.dart

```diff
@@ -35,15 +35,23 @@ void main() {
       );
 
       test(
-        'Loads the build metrics and shows the sparkline graph widgets',
+        'Loads the performance metric and shows it on sparkline graph',
```

Comment:
```suggestion
        "Loads the performance metric and shows it on sparkline graph",
```
Here and in other tests below. Could you please change to double quotes according to our discussions in this?